master/getting-started/kubernetes/installation/index.md

```diff
@@ -114,13 +114,22 @@ cat >/etc/cni/net.d/10-calico.conf <<EOF
         "type": "calico-ipam"
     },
     "policy": {
-        "type": "k8s"
+        "type": "k8s",
+        "k8s_api_root": "http://<K8S_API_IP>:<K8S_API_PORT>",
+        "k8s_auth_token": "<TOKEN>"
+    },
+    "kubernetes": {
+        "kubeconfig": "</PATH/TO/KUBECONFIG>"
     }
 }
 EOF
 ```
 
 Replace `<ETCD_IP>:<ETCD_PORT>` with your etcd configuration.
+Replace `<K8S_API_IP>:<K8S_API_PORT>` with your kube-apisever configuration.
```

Comment:
Hmm, so not all of these options are strictly required.

I'd recommend including the kubeconfig bit, and then linking to this doc for more information on the various configuration options:  https://github.com/projectcalico/calico-cni/blob/master/configuration.md#kubernetes-specific

WDYT?